README.md

```diff
@@ -1,2 +1,187 @@
-# js-signer
- This repository provides library to sign any arbitrary message with an EOSIO R1 key and produce an EOSIO signature
+# NodeJS/Javascript library for Signing Messages
+
+This repository provides library to sign any arbitrary message with an EOSIO R1 key and produce an EOSIO signature
```

Comment:
I wouldn't call this an `EOSIO` signature as they diverged and we're not in relation with them. Just say something along the lines of `Bullish compatible`